includes/helper/class-wp-job-manager-helper.php

```diff
@@ -624,7 +720,7 @@ private function handle_request() {
 	 * @return void
 	 */
 	private function handle_single_request() {
-		$licenced_plugins = $this->get_installed_plugins();
+		$licenced_plugins = $this->get_installed_plugins( false, false );
```

Comment:
Likewise, we should be allowing folks to active/deactivate licenses for inactive plugins.